frontend/src/stores/redux/store.ts

```diff
@@ -6,7 +6,7 @@ import rootReducer from "./reducers";
 const persistConfig = {
     key: 'root',
     storage,
-    whitelist: ["loggedUserState"],
+    whitelist: ["loggedUserState", "notificationState"],
```

Comment:
You show notification for some seconds. It does not makes sense to store it permanently